index.js

```diff
@@ -1343,6 +1344,7 @@ class GithubScm extends Scm {
                 lastCommitMessage: hoek.reach(webhookPayload, 'head_commit.message') || '',
                 hookId,
                 scmContext: scmContexts[0],
+                deleted,
```

Comment:
Why parseHook returns this object when deleted is `true`? Isn't it enough to return `null` when deleted is true?

Reply:
Thanks, I fixed